### DISCUSSION TITLE
Smart Connections links on properties and tags

### DISCUSSION THREAD

**Discussion by mijahauan** [brianpetro/obsidian-smart-connections#362] [General]
Perhaps I am missing something?  It seems that a markdown file augmented with a template will "connect" with ALL other files using that same (or similar) template -- even if they share nothing else in common!  This includes the "properties" section of tags, which Obsidian recognizes, but Smart Connections does not and so doesn't exclude it when looking for links.  Please point out my mistake or otherwise tell me why Smart Connections seems inimical to the use of templates!!

**Comment by GitTom** [comment]
I'm having a similar problem:  most of my notes start with 'created' and 'updated' properties and it always matches these.  So all of the top results in my 'Smart Connections Files' view are YAML property blocks.

Would be great if there was a "property_exclusions" option that allowed us to list properties to exclude.  Or maybe just exclude the entire YAML front matter if that is easier.
